src/server/closeDriver.js

```diff
@@ -0,0 +1,36 @@
+const psNode = require('ps-node');
+
+function kill(pid) {
+  return new Promise((resolve, reject) => {
+    psNode.kill(pid, (err) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve();
+      }
+    });
+  });
+}
+
+// Works around an issue in marionette causing ghost firefox instances to remain
+// running.
+//
+// https://bugzilla.mozilla.org/show_bug.cgi?id=1310992
+// https://github.com/mozilla/geckodriver/issues/285
+module.exports = function closeDriver(driver) {
+  return new Promise((resolve, reject) => {
+    driver.close();
+    psNode.lookup({
```

Comment:
Is there a way to get the pid from the `driver` object instead of doing this lookup?